api/infrastructure/parallelcluster-api.yaml

```diff
@@ -1191,7 +1192,7 @@ Resources:
                 Action:
                   - logs:CreateLogStream
                   - logs:PutLogEvents
-                Resource: !Sub arn:${AWS::Partition}:logs:${AWS::Region}:${AWS::AccountId}:log-group:/aws/lambda/ParallelClusterApi-UpdateParallelClusterLambda-*
+                Resource: !Sub arn:${AWS::Partition}:logs:${AWS::Region}:${AWS::AccountId}:log-group:/aws/lambda/*
```

Comment:
This unfortunately grants the Lambda permissions to write in any log group of any lambda function. One way to address this is to follow what we do in our cluster template, where we force the name of the lambda to be some prefix + a hash we get from the sack id. This way with a unique name we can scope down logs related permissions, given the fact that Lambda log groups names are based on the function name.

Also can you check if the same fix is required for the other Lambda present in this template?

Reply:
> Also can you check if the same fix is required for the other Lambda functions present in this template?

I'll look into this.